### ISSUE TITLE
[Update Request]: CondaForge.Mambaforge

### ISSUE THREAD

**Issue by NJT145** [microsoft/winget-pkgs#102049]
### What type of update are you requesting?

A new version of an existing package

### Current Package Identifier

CondaForge.Mambaforge

### Package Version

23.1.0-0

### Please describe the changes you would like to see

There is so many releases missing for that package.
Download page url: https://github.com/conda-forge/miniforge/releases

Check the [CondaForge.Miniforge3](https://github.com/microsoft/winget-pkgs/tree/master/manifests/c/CondaForge/Miniforge3) and update [CondaForge.Mambaforge](https://github.com/microsoft/winget-pkgs/tree/master/manifests/c/CondaForge/Mambaforge) for the same releases. Both must have same release versions at the end, I guess.

**Comment by BrandonWanHuanSheng** [comment]
Object has been enquired to Wget processing the link URL.

**Comment by NJT145** [comment]
> Object has been enquired to Wget processing the link URL.

What does that mean? Can you explain?

**Comment by BrandonWanHuanSheng** [comment]
> > Object has been enquired to Wget processing the link URL.
> 
> What does that mean? Can you explain?
Just ignore it.

**Comment by BrandonWanHuanSheng** [comment]
Right now, I just revoke the silent installer.

**Comment by BrandonWanHuanSheng** [comment]
Edited Some Command line from the previous. Migrated Successful. Need to Add /S, /JustMe, /AllUsers Remove /InstallerType. It is type installer of NullSoft. So there is a silent command.

**Comment by NJT145** [comment]
MambaForge releases still missing. Anyone working on that? Just the same repo and mamba instead of mini.
For example:
For version "23.1.0-0", MambaForge url is [https://github.com/conda-forge/miniforge/releases/download/23.1.0-0/Mambaforge-23.1.0-0-Windows-x86_64.exe](https://github.com/conda-forge/miniforge/releases/download/23.1.0-0/Mambaforge-23.1.0-0-Windows-x86_64.exe) .
For version "23.1.0-0", MiniForge url is [https://github.com/conda-forge/miniforge/releases/download/23.1.0-0/Miniforge3-23.1.0-0-Windows-x86_64.exe](https://github.com/conda-forge/miniforge/releases/download/23.1.0-0/Miniforge3-23.1.0-0-Windows-x86_64.exe) .
You see the similarity? Just the installar name is different. Same structure, just need to change "Miniforge3" with "Mambaforge".

**Comment by NJT145** [comment]
Also, same commands work on mambaforge...

**Comment by BrandonWanHuanSheng** [comment]
Waiting for it to process.
![Screenshot (190)](https://user-images.githubusercontent.com/118059864/231233522-9aed3f8e-b85d-4517-affe-4df2b16e8b29.png)

**Comment by BrandonWanHuanSheng** [comment]
Winget just handle one link at a time

**Comment by NJT145** [comment]
I found your pull request. It's there: #102220 . We need to do same for old versions too.

**Comment by BrandonWanHuanSheng** [comment]
Yes, I always tested the new custom switches. The old code switches it failed.

**Comment by NJT145** [comment]
There is a problem about update of Miniconda.
At admin powershell, it gives this error: `Installer failed with exit code: 2`.
And when I updated Miniforge3 with user powershell, it installs a second Miniforge at the same directory.
It also doesn't let an interactive installation.
We need to figure out what we can for that.
